pulp_rpm/app/advisory.py

```diff
@@ -190,6 +193,26 @@ def resolve_advisory_conflict(previous_advisory, added_advisory):
     return to_add, to_remove, to_exclude
 
 
+def _copy_update_collections_for(collections):
+    """
+    Deep-copy each UpdateCollection in the_collections, and its UpdateCollectionPackages.
+    """
+    new_collections = []
+    with transaction.atomic():
+        for collection in collections.prefetch_related('packages').all().iterator():
```

Comment:
I think `prefetch_related` doesn't work (just doesn't really prefetch) with `iterator` but @dralley knows better.

Reply:
Yup.  ```prefetch_related``` uses the cache, and ```.iterator()``` says "don't cache anything"

Reply:
Cool, I larned me yet another new thing, thanks - fixing